message-me/app/views/chatrooms/index.html.erb

```diff
@@ -12,11 +12,11 @@
         </div>
       </div>
       <div class="extra content">
-        <%= form_with class: 'ui reply form' do |form| %>
+        <%= form_with scope: :message, url: message_path, class: 'ui reply form' do |f| %>
           <div class="field">
             <div class="ui fluid icon input">
-              <i class="bordered inverted orange edit icon"></i>
-              <%= form.text_field :attr, placeholder: "Enter a message..." %>
+              <%= f.text_field :body, placeholder: "Enter a message..." %>
+              <%= f.button '<i class="bordered inverted orange edit icon"></i>'.html_safe %>
```

Comment:
use `f.submit` instead of button.

Why I ask you to do so?

Reply:
`f.submit` creates an `input type=submit`
![image](https://user-images.githubusercontent.com/67599130/126868595-56ef7f90-ad4d-449f-b14b-d327ca505557.png)

While,
`f.button` creates `button type=submit`
![image](https://user-images.githubusercontent.com/67599130/126868617-a8311465-d68f-4545-b910-6a269b8f3097.png)

the `f.submit` dont work with `.html_safe` ,  and It generates `input type='submit'`, so it cant display the icon. But then `f.button generates `button type='submit'` and can add icon. Is it still better to use `f.submit` in our case where we need to use icon?